### DISCUSSION TITLE
Add menu item to force FLASH GarbaceCollectot

### DISCUSSION THREAD

**Discussion by RedzIta** [fedorovvl/tso_client#412] [General]
Is it possible to add a new menu item "Execute FLASH Garbage Collector"? That's can be remove some "memory leak".

**Comment by fedorovvl** [comment]
![изображение](https://github.com/user-attachments/assets/64debe05-6435-45f7-ab1f-ee1454801bf5)
this setting execute System.gc() every 60 sec

**Reply by RedzIta** [reply]
Ok, thanks

**Comment by fedorovvl** [comment]
if u rly need menu item then use this userscript
```
addToolsMenuItem("Execute FLASH Garbage Collector", function(event) { window.runtime.flash.system.System.gc(); });
```
